src/DriverFactory.php

```diff
@@ -40,6 +40,7 @@ protected function createWithConfig(ContainerInterface $container, $configKey)
         }
 
         if (AnnotationDriver::class === $config['class'] || is_subclass_of($config['class'], AnnotationDriver::class)) {
+            // @todo $config['cache'] needs to be an instance currently
```

Comment:
`$container->get('array');` for example isn't going to work (currently works in DoctrineModule... not sure how compatible you want to be?)

IMO it's fine to do this, but it needs to be documented; your call :)